packages/@tinacms/git-client/src/git-file.ts

```diff
@@ -36,17 +36,19 @@ export class GitFile {
   }
 
   commit = () => {
-    return this.git.commit!({
+    return this.cms.api.git.commit!({
       files: [this.relativePath],
-    })
+    }).then((response: Response) =>
+      this.cms.events.dispatch({ type: 'git:commit', response })
+    )
   }
 
   reset = () => {
     this.cms.api.git.reset({ files: [this.relativePath] })
   }
 
   write = (values: any) => {
-    this.git.writeToDisk!({
+    this.cms.api.git.writeToDisk!({
```

Comment:
What prompted this switch?

Reply:
Oh I was having trouble testing with `this.git`...commits weren't working so I thought it may have been a bug, but after switching back and rebuilding it seems to work fine. I'll change that.